This is too long

warp-core opened the pull request.

MasterOfGrey commented:
Just so you know, there's a lot of stuff in this. Most of it looks like stylistic preference, and precisely 0 explanation has been given justifying any of it.

I will continue to merge any PR's that are clear fixes, or pull out anything that is obvious at a glance from combination PR's.

However, without a full explanation as to what's going on here and why, it's not going to happen until I have time to review the whole thing line by line - which isn't happening any time soon.

warp-core commented:
A decent chunk of the changes I had made you've already done with the latter missions (putting `Hai Reveal` at the start).

There's no need to put `ZZ` at the start of missions that offer immediately after the First Contact: `H` is already after `F`, and as I've said, those missions should be replaced with total overrides of the first contacts at some point (but I accept that that isn't necessary for this to work).

The initial missions that start this all off *should* be minor. That is what it is for, and there's no good reason to block any minor missions: the FW intro missions don't.

I've moved the event from file 3 to 1 because it is used in 1, I don't know why Derpy would tell you to put it in 3 unless he meant that it shouldn't be used until 3.

I've moved initial planet definitions for Mountaintop and Mooncrack out of events because there is no reason to put them in events. The planets are not used anywhere else so there is no reason to not have them exist, that just adds clutter to the savefile. While the game is not particularly averse to doing this, I don't think we should do it when it is totally unnecessary.

Similarly, I've moved the Mountaintop and Mooncrack objects to orbiting invisible objects so that they can initially be `add`ed instead of redefining the whole system (which not only adds to the savefile but also risks making mistakes, and is another thing to update if, for example, sprites are changes, Cognizance has already had these issues occur because it redefines, like, 5 systems) and, while the additional redefinitions are necessary, putting these objects around invisible objects means it'll be easier to `remove` them (once that feature is added, either by my PR or another) while not making any difference to the way things are right now.
